internal/task/scripts/balance.go

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2022-04-22
+ * Author: Ye Tao (YhhHaa)
+ */
+
+package scripts
+
+// part of deploying used to balance curvebs cluster
+var BALANCE = `
+#!/bin/bash
```

Comment:
you can let `/usr/bin/env` to help you find the path of `bash`, it can makes scripts more portable, like:

```
#!/usr/bin/env bash
```